x/harvest/keeper/keeper.go

```diff
@@ -114,11 +114,10 @@ func (k Keeper) IterateDeposits(ctx sdk.Context, cb func(deposit types.Deposit)
 	}
 }
 
-// IterateDepositsByTypeAndDenom iterates over all deposit objects in the store with the matching deposit type and deposit denom and performs a callback function
-func (k Keeper) IterateDepositsByTypeAndDenom(ctx sdk.Context, depositType types.DepositType, depositDenom string, cb func(deposit types.Deposit) (stop bool)) {
-
+// IterateDepositsByDenom iterates over all deposit objects in the store with the matching deposit denom and performs a callback function
+func (k Keeper) IterateDepositsByDenom(ctx sdk.Context, depositDenom string, cb func(deposit types.Deposit) (stop bool)) {
```

Comment:
Add `GetTotalDeposited`, `SetTotalDeposited`, `IncrementTotalDeposited`, and `DecrementTotalDeposited` methods

Reply:
See https://github.com/Kava-Labs/kava/pull/716#discussion_r522122035